create-repo

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+# vim: ft=sh sw=2 et
+# shellcheck shell=bash
+set -efo pipefail
+
+# A Bash script that automates creating a Docker Hub Repository, if applicable. Also adds an org group, engineering, to have read/write access to said repo if applicable
+
+#### Requires the following parameters to be passed in from the CLI:
+# Docker Hub API Token = $1
+# User/Org name = $2
+# Repo name = $3
+
+# Checks if required values are passed in
+test -z $1 && (echo "Script needs a token towards Docker Hub!" && exit 1)
+test -z $2 && (echo "Script needs a user/org that exists in Docker Hub!" && exit 1)
+test -z $3 && (echo "Script needs a repo name to verify/use!" && exit 1)
+
+REPO_EXISTS_CODE=$(curl --write-out %{response_code} --output /dev/null -s \
+    -H "Content-Type: application/json" -H "Authorization: JWT $1" \
+    --url https://hub.docker.com/v2/repositories/$2/$3/)
+
+if [ $REPO_EXISTS_CODE -ne 200 ]; then
+    echo "Docker Hub repo $3 does not exist in $2, creating now..."
+    cat << EOF > ./data.json
```

Comment:
If we are going to write out this data.json can we just put it in a temp directory just in case? Id hate for this to end up clobbering something else in the future.

```
tmpdata=$(mktemp /tmp/tempdata.XXXXXX)
cat << EOF > $tmpdata
...
EOF
```

Reply:
Good catch on that! I think that'll help keep things more consistent within the GitHub Workflows.